jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/PropertiesExtender.java

```diff
@@ -0,0 +1,16 @@
+package org.eclipse.jkube.kit.common;
+
+import java.net.URL;
+import java.util.Properties;
+
+public class PropertiesExtender extends Properties {
```

Comment:
I'm not sure about the motivation behind using this POJO. Why not use a `URL` field directly?

Reply:
Hello,
Sorry for delayed response, I was preparing for certification exam.
`java.util.Properties` class is returned by `SpringBootUtil.getSpringBootApplicationProperties()`, however that class doesn't have necessary fields which might be used for storing URL/File object so I decided to extend it to bring properties file location together with its properties list:

```
public static SpringBootConfiguration from(JavaProject project) {
    final PropertiesExtender properties = SpringBootUtil.getSpringBootApplicationProperties(
      SpringBootUtil.getSpringBootActiveProfile(project),
      JKubeProjectUtil.getClassLoader(project));
     project.setPropertiesFile(properties.getPropertiesFile());
```